### PR TITLE
Adding_quiz_link

### PR DESCRIPTION
**Pull request by pritamps**
Also deleted some unused routes

### CODE REVIEW

File: app/routers/reports.py [hunk 1]
```diff
@@ -18,6 +18,11 @@
     "rank": "Rank",
 }
 
+QUIZ_URL = (
+    "https://quiz.avantifellows.org/quiz/{quiz_id}?userId={user_id}&apiKey={api_key}"
+)
+AF_API_KEY = "6qOO8UdF1EGxLgzwIbQN"
```
Comment (sudheshna-donthineni) [inline app/routers/reports.py:24]: Is it okay to leave this hardcoded?

Comment (pritamps) [reply app/routers/reports.py:24]: No it's okay. It's in all our quiz URLs, no point hiding it :P

**Review by sudheshna-donthineni — APPROVED** [review]
Looks fine to me!

**Review by suryabulusu — APPROVED** [review]
looks good!

**Pull request merged**